src/logsearch-filters-common/target/logstash-filters-default.conf.erb

```diff
@@ -0,0 +1,128 @@
+if [@type] in ["syslog", "relp"] {
```

Comment:
I'm not a huge fan of having two compiled files committed. On the one hand, this is a static file, so it's much more appropriate to be included in the logstash package, bypassing the ERB double-escaping rationale for this particular file. On the other hand, I do feel like we might have a need to make something here user-configurable (e.g. default timezone).